src/Npgsql/NpgsqlBinaryExporter.cs

```diff
@@ -106,7 +106,7 @@ void ReadHeader()
         /// The number of columns in the row. -1 if there are no further rows.
         /// Note: This will currently be the same value for all rows, but this may change in the future.
         /// </returns>
-        public int StartRow() => StartRow(false).GetAwaiter().GetResult();
+        public int StartRow() => StartRow(false, default).GetAwaiter().GetResult();
```

Comment:
Should not be needed - same for all other cases where `default` was added for cancellation token